### PR TITLE
README updated and minor refactorings

### PR DESCRIPTION
**Pull request by joaodias14**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,137 @@
...
+
+## Create the Admin server
+To create the Admin server you just need to build a regular Spring Boot app with the following details:
+
+ - Add the dependency `spring-boot-admin-starter-server` to the `pom.xml`;
+```
+<dependency>
+	<groupId>de.codecentric</groupId>
+	<artifactId>spring-boot-admin-starter-server</artifactId>
+</dependency>
```
Comment (bsferreira) [inline README.md]: we also need spring-boot-admin-dependencies here

File: README.md [hunk 2]
```diff
@@ -1,2 +1,137 @@
...
+### App instance HTTP Traces
+In the `Http Traces` tab you can check detailed information about each HTTP request to the App instance. As an example, perform the following steps:
+
+ 1. In the Spring Boot Admin dashboard select the instance of the `Table Tenis` app you are about to call;
+ 2. Go to the `HttpTraces` tab;
+ 3. Call one of the endpoints exposed by the `Table Tenis` app, e.g. http://localhost:50345/ping;
+ 4. Check that a new request is displayed in the `HttpTrances` tab (you can access the request details by clicking on it).
+
+### App instance Auditing
+TODO
```
Comment (bsferreira) [inline README.md]: This is only available when security is enabled (response from SBA author)

File: admin-client-autodb/src/main/java/io/mercedesbenz/springio/adminclientautodb/ScheduledTaskExample.java [hunk 3]
```diff
@@ -12,7 +12,7 @@ public void ping(){
     }
 
     @Scheduled(fixedRate=1000)
-    public void ponng(){
+    public void pong(){
```
Comment (bsferreira) [inline admin-client-autodb/src/main/java/io/mercedesbenz/springio/adminclientautodb/ScheduledTaskExample.java]: Maybe we should change this to a different thing to not create confusion with the other app. Maybe we can just call it builtCars() and print the a number that is always increasing? what do you say? I can change that if you want

File: admin-client-autodb/src/main/java/io/mercedesbenz/springio/adminclientautodb/TasksController.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    private ThreadPoolTaskScheduler taskScheduler;
+
+    @PostMapping(value = "/create")
+    @ResponseStatus(HttpStatus.CREATED)
+    public void createNewScheduledTask() {
+        LocalDateTime dateTime = LocalDateTime.now().plusMinutes(10);
+        this.taskScheduler.schedule(() -> System.out.println("Task running"), dateTime.toInstant(ZoneOffset.UTC));
+    }
+
+}
```
Comment (bsferreira) [inline admin-client-autodb/src/main/java/io/mercedesbenz/springio/adminclientautodb/TasksController.java]: What's this for?

File: README.md (hunk 1 above)
Comment (joaodias14) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (joaodias14) [reply README.md]: Removed

File: admin-client-autodb/src/main/java/io/mercedesbenz/springio/adminclientautodb/ScheduledTaskExample.java (hunk 3 above)
Comment (joaodias14) [reply admin-client-autodb/src/main/java/io/mercedesbenz/springio/adminclientautodb/ScheduledTaskExample.java]: Changed

**Pull request merged**